CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ### Enhancements
 
+* Bundled xcube-viewer 1.3.0.
```

Comment:
link it to the release tag. See other bullet points regarding 'Bundled xcube-viewer' in `CHANGES.md`